draft-ietf-httpbis-encryption-encoding.md

```diff
@@ -598,6 +629,27 @@ added for presentation purposes only.
 ~~~
 
 
+## Diffie-Hellman with Authentication Secret
+
+This example shows the same receiver key pair from {{ex-dh}}, but with a shared
+authentication secret of "R29vIGdvbyBnJyBqb29iIQ".
+
+~~~
+HTTP/1.1 200 OK
+Content-Length: 33
+Content-Encoding: aesgcm
+Encryption: keyid="dhkey"; salt="lngarbyKfMoi9Z75xYXmkg"
+Crypto-Key: keyid="dhkey";
+                dh="BNoRDbb84JGm8g5Z5CFxurSqsXWJ11ItfXEWYVLE85Y7
+                    CYkDjXsIEc4aqxYaQ1G8BqkXCJ6DPpDrWtdWj_mugHU"
+
+QPsWCEH-8WqiXl7Ez23A6u4rwH2hFBifCybQQz-SxsjF
```

Comment:
I am unable to reproduce this in either of my implementations, using the following information:

**Sender public**: BNoRDbb84JGm8g5Z5CFxurSqsXWJ11ItfXEWYVLE85Y7CYkDjXsIEc4aqxYaQ1G8BqkXCJ6DPpDrWtdWj_mugHU
**Sender private**: nCScek-QpEjmOOlT-rQ38nZzvdPlqa00Zy0i6m2OJvY
**Receiver public**: BCEkBjzL8Z3C-oi2Q7oE5t2Np-p7osjGLg93qUP0wvqRT21EEWyf0cQDQcakQMqz4hQKYOQ3il2nNZct4HgAUQU
**Salt**: lngarbyKfMoi9Z75xYXmkg
**Auth secret**: R29vIGdvbyBnJyBqb29iIQ
**Padding**: 0 bytes
**Plaintext**: I am the walrus

I get the following:

**Shared secret**: RNjC-NVW4BGJbxWPW7G2mowsLeDa53LYKYm4--NOQ6Y
**PRK**: EhpZec37Ptm4IRD5-jtZ0q6r1iK5vYmY1tZwtN8fbZY
**CEK**: AN2-xhvFWeYh5z0fcDu0Ww
**Nonce**: JY1Okw5rw1Drkg9J
**Ciphertext**: 6nqAQUME8hNqw5J3kl8cpVVJylXKYqZOeseZG8UueKpA

Decryption fails as well, so I can't see whether you used a different `plaintext`.